pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -547,6 +563,17 @@ func (asw *actualStateOfWorld) AddPodToVolume(markVolumeOpts operationexecutor.M
 		podObj.mounter = mounter
 	}
 	asw.attachedVolumes[volumeName].mountedPods[podName] = podObj
+	if utilfeature.DefaultFeatureGate.Enabled(features.ExpandInUsePersistentVolumes) {
```

Comment:
This is using `ExpandInUsePersistentVolumes` feature gate.

Reply:
Fixed